src/components/valet-reservation/ReservationList.jsx

```diff
@@ -1,20 +1,64 @@
-
+import { useState, useEffect } from "react";
+import getReservationList from "../../utils/getReservationList";
 const ReservationList = () => {
-  // const { reservations } = useReservationContext();
+  const [reservations, setReservations] = useState([]);
+  const [currentHour, setCurrentHour] = useState(new Date().getHours());
+  const[showNextHour, setShowNextHour] = useState(false);
+
+  const fetchReservations = async () => {
+    const garage_id = 1;
```

Comment:
Looks like these are hard coded for testing purposes right now, I think that's what you mentioned on our call. Good for now!